db/schema.rb

```diff
@@ -192,7 +192,7 @@
     t.text "strengths"
     t.text "skills"
     t.text "interests"
-    t.string "state", default: "interested/registered"
+    t.string "state", default: "registered"
```

Comment:
Yes, but it is generated through the migrations. Travis currently just loads the schema, but when someone runs db:migrate, this line of the schema will change again.